### DISCUSSION TITLE
Adding units to processed bids

### DISCUSSION THREAD

**Discussion by Raeggincom** [UNSW-CEEM/nempy#10] [Q&A] [answered]
Hi Nick!

Great job on the NEMPY package!

If I would like to add a unit to the 'price_bids' and 'volume_bids', how do I go about it?

I tried appending a unit to the processed bids but I receive the following error 'The column unit can only contain the values 0' when trying to run the sample code you provided

Cheers,
Raeggin

**Comment by Raeggincom** [comment]
Apologies, the error message is as such 'The column unit can only contain the values' followed by a list of units- when trying to run the line: market.set_unit_volume_bids(volume_bids)

**Comment by nick-gorman** [comment, marked as answer]
Hi @Raeggincom,

Have you also added the unit to the unit_info table? Nempy checks that all the units provided in the bidding tables are also in the unit_info table, and throws the error you are getting if they're missing.

**Reply by Raeggincom** [reply]
Thank you! That fixed it.
